packages/lisk-transactions/test/transactions/base.ts

```diff
@@ -17,647 +17,498 @@ import { BaseTransaction } from '../../src/transactions/base';
+			expect(baseTransaction)
 				.to.have.property('senderPublicKey')
 				.and.be.a('string');
 		});
 
-		it('should have signature string', () => {
-			return expect(baseTransaction)
+		it('should have signature string', async () => {
+			expect(baseTransaction)
 				.to.have.property('senderPublicKey')
 				.and.be.a('string');
 		});
 
-		it('should have signSignature string', () => {
-			return expect(baseTransaction)
+		it('should have signSignature string', async () => {
+			expect(baseTransaction)
 				.to.have.property('senderPublicKey')
 				.and.be.a('string');
 		});
 
-		it('should have signatures array', () => {
-			return expect(baseTransaction)
+		it('should have signatures array', async () => {
+			expect(baseTransaction)
 				.to.have.property('signatures')
 				.and.be.a('array');
 		});
 
-		it('should have timestamp number', () => {
-			return expect(baseTransaction)
+		it('should have timestamp number', async () => {
+			expect(baseTransaction)
 				.to.have.property('timestamp')
 				.and.be.a('number');
 		});
 
-		it('should have type number', () => {
-			return expect(baseTransaction)
+		it('should have type number', async () => {
+			expect(baseTransaction)
 				.to.have.property('type')
 				.and.be.a('number');
 		});
 
-		it('should have receivedAt Date', () => {
-			return expect(baseTransaction)
+		it('should have receivedAt Date', async () => {
+			expect(baseTransaction)
 				.to.have.property('type')
 				.and.be.a('number');
 		});
 
-		it('should have isMultisignature boolean', () => {
-			return expect(baseTransaction)
+		it('should have isMultisignature boolean', async () => {
+			expect(baseTransaction)
 				.to.have.property('isMultisignature')
 				.and.be.a('boolean');
 		});
 
-		describe('when transaction has empty `signatures`', () => {
-			it('should set isMultisignature to false', () => {
-				return expect(baseTransaction.isMultisignature).to.be.false;
-			});
+		it('should set isMultisignature to true with a multisignature transaction', async () => {
+			const multisignatureTestTransaction = new TestTransaction(
+				defaultMultisignatureTransaction,
+			);
+			expect(multisignatureTestTransaction.isMultisignature).to.be.true;
 		});
 
-		describe('when transaction has non-empty `signatures`', () => {
-			beforeEach(() => {
-				const multisignatureTransaction = {
-					...defaultTransaction,
-					signatures: [defaultSignature, defaultSignature],
-				};
-				baseTransaction = new TestTransaction(multisignatureTransaction);
-			});
-			it('should set isMultisignature to true', () => {
-				return expect(baseTransaction.isMultisignature).to.be.true;
-			});
+		it('should set isMultisignature to false with non-multisignature transaction', async () => {
+			expect(baseTransaction.isMultisignature).to.be.false;
 		});
 
-		describe('when given a transaction with invalid types', () => {
+		it('should throw a transaction multierror with an invalid typed transaction', async () => {
```

Comment:
The description is maybe incorrect here.

Reply:
how so? `incorrectly typed` is better?